docs/examples/registry/hpa.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: core.oam.dev/v1alpha2
+kind: TraitDefinition
+metadata:
+  annotations:
+    definition.oam.dev/description: "configure k8s HPA for Deployment"
+  name: hpa
+spec:
+  appliesToWorkloads:
+    - webservice
+    - worker
+  schematic:
+    cue:
+      template: |
+        outputs: hpa: {
+        	apiVersion: "autoscaling/v2beta2"
+        	kind:       "HorizontalPodAutoscaler"
+        	metadata: name: context.name
+        	spec: {
+        		scaleTargetRef: {
```

Comment:
hard code this in the defnition?

Reply:
yes， it's apply to Deployment kind workload.